package.json

```diff
@@ -20,29 +20,30 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "^4.9.2",
-    "@typechain/ethers-v5": "^11.0.0",
-    "@typechain/hardhat": "^8.0.0",
+    "@opengsn/contracts": "^3.0.0-beta.6",
```

Comment:
Are we sure about this? Them being in beta means not audited yet - no?
I'd either wait for `3.0.0` to be a thing, or use `2.0.0` unless there are major changes between the two.
We could discuss this one.

Reply:
For example... They're already at `3.0.0-beta.9`. I'd be a bit scared to go for a production deploy with a beta.